src/test/java/com/camacho/simpleannotation/builders/tests/FakeClassHappyPathTest.java

```diff
@@ -13,7 +14,7 @@ final class FakeClassHappyPathTest {
     void buider_mustHave_necessaryMethods () {
         List<String> expectedMethods = Arrays.asList("setI", "setD", "setS", "setStrings", "build");
 
-        assertTrue(Arrays.stream(FakeClassHappyPathBuilder.class.getMethods())
+        assertTrue(Arrays.stream(com.camacho.simpleannotation.builders.FakeClassHappyPathBuilder.class.getMethods())
```

Comment:
It seems that there are no name collisions with FakeClassHappyPathBuilder.class. Consider adding an import statement

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)